packages/p2p/src/components/my-ads/ad-status.scss

```diff
@@ -4,11 +4,7 @@
         border-radius: 1.6rem;
         padding: 0.2rem 1.6rem;
         text-align: center;
-        width: 8.8rem;
-
-        @include mobile {
-            margin-bottom: 0.8rem;
-        }
+        width: 7.4rem;
```

Comment:
Did you check responsive? Why was the responsive style removed?

Reply:
Pulled it from Artem's changes... Will revert it